### ISSUE TITLE
An incompatible architecture (have (arm64), need (x86_64h))

### ISSUE THREAD

**Issue by asyschikov** [reloadware/reloadium#83]
## Describe the bug*
Installed reloadium, on the first debug attempt I get the error:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.10/3.10.6_2/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 187, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/local/Cellar/python@3.10/3.10.6_2/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 146, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/local/Cellar/python@3.10/3.10.6_2/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 110, in _get_module_details
    __import__(pkg_name)
  File "/Users/syschiko/.reloadium/package/3.10/reloadium/__init__.py", line 4, in <module>
    pre_import_check()
  File "/Users/syschiko/.reloadium/package/3.10/reloadium/__utils__.py", line 21, in pre_import_check
    import reloadium.corium
ImportError: dlopen(/Users/syschiko/.reloadium/package/3.10/reloadium/corium.cpython-310-darwin.so, 0x0002): tried: '/Users/syschiko/.reloadium/package/3.10/reloadium/corium.cpython-310-darwin.so' (mach-o file, but is an incompatible architecture (have (arm64), need (x86_64h)))
```

This is Intel, not M1 Mac.

## To Reproduce
Kinda not sure.

## Expected behavior
At least not to crash on the start.

## Desktop (please complete the following information):**
 - OS: MacOS
 - OS version: 12.6.2
 - M1 chip: no
 - Reloadium package version: 0.9.1
 - PyCharm plugin version: 0.9.1
 - Editor: PyCharm
 - Python Version: 3.10
 - Python Architecture: 64
 - Run mode: Debug

```
PyCharm 2022.3.1 (Professional Edition)
Build #PY-223.8214.51, built on December 20, 2022
Runtime version: 17.0.5+1-b653.23 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.6.2
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 12
Metal Rendering is ON
Registry:
    debugger.watches.in.variables=false

Non-Bundled Plugins:
    org.reloadium (0.9.1)
```

**Comment by dkrystki** [comment]
Fixed in reloadium 0.9.7 and PyCharm plugin 0.9.2

**Comment by asyschikov** [comment]
It works, thanks!
